HW03/maksim_manzhai/spec/maksim_manzhai_hw_03_t_01_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require_relative '../maksim_manzhai_hw_03_t_01'
+
+RSpec.describe Homework3 do
+  describe '#task1' do
+    let(:homework) { described_class.new }
```

Comment:
you can do it this way

```suggestion
    subject { described_class.new.task1(log) }
```

Reply:
try to apply it to all tests